frontend/src/components/RotateCredentials.vue

```diff
@@ -0,0 +1,469 @@
+          successMessage: `Rotation of etcd encryption key initiated for ${this.shootName}`,
+          heading: 'Do you want to initiate the rotation of etcd encryption key?',
+          alert: {
+            type: 'info',
+            messages: [
+              'A new encryption key will be created and added to the bundle (together with the old encryption key). All Secrets in the cluster will be rewritten by the kube-apiserver so that they become encrypted with the new encryption key.'
+            ]
+          }
+        },
+        'rotate-etcd-encryption-key-complete': {
+          caption: this.isLoading
+            ? 'Completing etcd encryption key rotation'
+            : this.isDisabled
+              ? 'Rotation initiation not completed'
+              : 'Complete ETCD Encryption Key Rotation',
+          errorMessage: 'Could not complete the rotation of etcd encryption key',
+          successMessage: `Rotation of etcd encryption key completed for ${this.shootName}`,
+          heading: 'Do you want to complete the rotation of etcd encryption key?',
+          alert: {
+            type: 'warning',
+            messages: [
+              'The old encryption will be dropped from the bundle.'
+            ]
+          }
+        },
+        'rotate-serviceaccount-key-start': {
+          caption: this.isLoading
+            ? 'Initiating ServiceAccount token signing key rotation'
+            : this.isDisabled
+              ? 'Rotation already initiated'
+              : 'Initiate ServiceAccount Token Signing Key Rotation',
+          errorMessage: 'Could not initiate the rotation of ServiceAccount token signing key',
+          successMessage: `Rotation of ServiceAccount token signing key initiated for ${this.shootName}`,
+          heading: 'Do you want to initiate the rotation of ServiceAccount token signing key?',
+          alert: {
+            type: 'info',
+            messages: [
+              'A new signing key will be created and added to the bundle (together with the old signing key)'
+            ]
+          }
+        },
+        'rotate-serviceaccount-key-complete': {
+          caption: this.isLoading
+            ? 'Completing ServiceAccount token signing key rotation'
+            : this.isDisabled
+              ? 'Rotation initiation not completed'
+              : 'Complete ServiceAccount Token Signing Key Rotation',
+          errorMessage: 'Could not complete the rotation of ServiceAccount token signing key',
+          successMessage: `Rotation of ServiceAccount token signing key completed for ${this.shootName}`,
+          heading: 'Do you want to complete the rotation of ServiceAccount token signing key?',
+          alert: {
+            type: 'warning',
+            messages: [
+              'Ensure that end-users have updated all out-of-cluster API clients that communicate with the control plane via ServiceAccount tokens. The old signing key will be dropped from the bundle.'
```

Comment:
if we take the suggestion above, we could adapt this text accordingly as well

Reply:
See comment above

Reply:
done